python/dgl/distributed/partition.py

```diff
@@ -258,26 +383,115 @@ def partition_graph(g, graph_name, num_parts, out_path, num_hops=1, part_method=
     >>> g, node_feats, edge_feats, gpb, graph_name = dgl.distributed.load_partition(
     ...                                 'output/test.json', 0)
     '''
+    def get_homogeneous(g, balance_ntypes):
```

Comment:
Any particular reason to put this inside a function?

Reply:
no particular reasons. this function is just very specific to `partition_graphs`.

Reply:
I would move this outside the function and make it private (prefix with an underscore).  Otherwise the function definition will be run every time you call it.  Although that may not be a big deal.